templates/nas-grp-base.html.twig

```diff
@@ -0,0 +1,150 @@
+            function gtag(){dataLayer.push(arguments);}
+            gtag('js', new Date());
+
+            gtag('config', '{{google_analytics_id}}', {
+                'page_path': '{{ app.request.pathinfo }}'
+            });
+        </script>
+
+    {% block stylesheets %}
+        {{ encore_entry_link_tags('nas-app') }}
+    <link href="//cdnjs.cloudflare.com/ajax/libs/qtip2/2.2.0/jquery.qtip.min.css" rel="stylesheet" />
+
+        {% if isMaintenanceMode() %}
+    <link href="{{ asset('css/griidc_site_banner_screen.css') }}" rel="stylesheet" />
+        {% endif %}
+    {% endblock stylesheets  %}
+    {% endblock %}
+</head>
+<body class="html page-pelagos-full">
+{% block htmlbody %}
+    <ul class="atlassian-btns">
+        <li>
+            <a href="#" id="atlwdg-issue" class="atlwdg-trigger">
+                Report Issue
+            </a>
+        </li>
+        <li>
+            <a href="#" id="atlwdg-suggest" class="atlwdg-trigger">
+                Suggest Improvement
+            </a>
+        </li>
+    </ul>
+    <div id="page-wrapper">
+        <div id="page">
+            {% if isMaintenanceMode() %}
+                <div id="siteBannerHeaderBanner" class="toolbar clearfix toolbar-processed overlay-displace-top" style="background-color:{{ getMaintenanceModeColor() }}">
+                    {{ getMaintenanceModeText()}}
+                </div>
+                <div id="siteBannerFooterBanner" class="toolbar clearfix toolbar-processed overlay-displace-top" style="background-color:{{ getMaintenanceModeColor() }}">
+                    {{ getMaintenanceModeText()}}.</div>
+            {% endif %}
+            <div id="header"><div class="section clearfix">
+
+                    <div class="region region-header">
+                        <div id="block-pelagos-pelagos-login-logout-links" class="block block-pelagos">
+                            <div class="content">
+                                {% if is_granted('ROLE_USER') %}
+                                    <a style="text-decoration: underline;">
+                                        Logged in as: {{ app.user.username }}
+                                    </a>
+                                    |
+                                    <a href="{{ path('security_logout') }}">  Log out</a>
+                                {% else %}
+                                    <a href="{{ path('security_login', {destination: app.request.uri}) }}">Log in</a>
+                                {% endif %}
+                            </div>
+                        </div>
+                    </div>
+
+                    <div id="branding" class="container">
+                        <div id="logo-wrapper">
+                            <a href="http://gulfresearchinitiative.org" title="National Academies | Gulf Research Program" rel="home" id="logo">
+                                <img src="{{asset('images/nas-grp-logo.png')}}" alt="Home" style="width: 233px; height: 80px">
```

Comment:
This image is a bit squeezed.

Reply:
Do you want to merge it to your branch? I think you've fixed styling right